demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -2255,6 +2280,25 @@ export class DemoMeetingApp
     });
   }
 
+  async startLiveConnector(): Promise<any> {
```

Comment:
This is 4 space indentation seems like: Can you indent this with two space?

Reply:
just checked. It has the same indentation as others, which are two space.